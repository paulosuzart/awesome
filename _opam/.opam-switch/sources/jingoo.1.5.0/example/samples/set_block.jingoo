{%- set list -%}
{%- macro li (content) %}<li>{{ content }}</li>{% endmacro -%}
{%- set item4 = '<Escaping works>' -%}
<ol>
  {{ li ('Item1') }}
  {{ li ('Item2') }}
  {{ li ('Item3') }}
  {{ li (item4) }}
</ol>
{%- endset -%}

Stored output:
{{ list }}

Variables defined inside block are not exposed:
{{ item4 }}

Stored output again:
{{ list }}
