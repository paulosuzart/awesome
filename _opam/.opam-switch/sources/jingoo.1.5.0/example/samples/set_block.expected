Stored output:
<ol>
  <li>Item1</li>
  <li>Item2</li>
  <li>Item3</li>
  <li>&lt;Escaping works&gt;</li>
</ol>

Variables defined inside block are not exposed:


Stored output again:
<ol>
  <li>Item1</li>
  <li>Item2</li>
  <li>Item3</li>
  <li>&lt;Escaping works&gt;</li>
</ol>
